README.md

```diff
@@ -84,7 +84,7 @@ Due to an issue with ESLint, config extension cannot be called from a globally i
 - [ESLint](https://github.com/eslint/eslint) 2.10.2
 - [eslint-plugin-react](https://www.npmjs.com/package/eslint-plugin-react) 6.0.0
 - [eslint-plugin-filenames](https://www.npmjs.com/package/eslint-plugin-filenames) 1.1.0
-- [eslint-plugin-import](https://www.npmjs.com/package/eslint-plugin-import) 1.12.0
+- [eslint-plugin-import](https://www.npmjs.com/package/eslint-plugin-import) 1.13.0
```

Comment:
Right. In this case there are no dependencies being updated, and nothing functional changing. We're just fixing a mismatch as the config is aligned with `1.13.0` - if you'd like I can move this readme change into a separate pull request?